cluster/get-kube-local.sh

```diff
@@ -76,12 +75,11 @@ function create_cluster {
   --privileged=true \
   -d \
   gcr.io/google_containers/hyperkube-${arch}:${release} \
-    /hyperkube kubelet \
-      --containerized \
+    /kubelet-runner.sh \
       --hostname-override="127.0.0.1" \
       --address="0.0.0.0" \
       --api-servers=http://localhost:8080 \
-      --config=/etc/kubernetes/manifests \
+      --config=etc/kubernetes/manifests \
```

Comment:
@errordeveloper It's tricky but it makes sense. After executing nsenter into the host's namespace, `/` refers to the host's root directory (in which /etc/kubernetes/manifests may not exist). However, the CWD (`.`), which doesn't change with the mount namespace, still has access to the container's filesystem letting you access to the container's file.